sampleReact.jsx

```
@@ -0,0 +1,11 @@
+function TestComponent() {
+    // ruleid:react-dangerouslysetinnerhtml
+    return <div dangerouslySetInnerHTML={createMarkup()} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)